skynet/bandwidth.go

```diff
@@ -11,6 +11,20 @@ const (
 	// SizeChunk is the size of a chunk.
 	SizeChunk = 40 * MiB
```

Comment:
Why is a chunk 40mb?

Danger was confused that a 4.37 MB file is reported at 55.9 MB, since the redundancy for large files should be less than for small ones. I'm guessing it's because chunks are so large, but that really should be explained somewhere on the site, it's quite confusing (even for me).

Danger's tests:

1) 3.88 MB file > 14 MB
2) 4.37 MB file > 55.9 MB

Reply:
We have two data packages we work with - sector (4 MB) and chunk (10 sectors, i.e. 40 MB). I know there were plans for a smaller package called segment (64 byte) but I don't know what happened to it.

Each Sia file takes at least one sector which we call the base sector. That sector contains the basic metadata of the sia file and as much of the content as we can manage. This sector is very important and it's uploaded with 10x redundancy. Any data that won't fit in these 4 MB overflows in one or more chunks. Chunks are 40 MB each and are uploaded with 3x redundancy. The reason they are exactly 40 MB is that each chunk contains 10 sectors and 3x redundancy comes from the Reed-Salomon error correction encoding which translates those 10 sectors into 40 resulting sectors, any 10 of which are sufficient for reconstructing the original chunk data.

Also, we are currently reporting the size of skyfiles as raw size / 3 which is not very intuitive, so we are backtracking on that. But that's the reason you're currently seeing a 3MB file take 40/3 = 13.3MB of space and a 4.1 MB file take (4*10 + 40*3) / 3 = 53MB.